wiki/People/osu!_team/en.md

```diff
@@ -11,29 +11,39 @@ tags:
 
 The people listed below are the core of the **osu! team** and are chiefly responsible for helping make the game awesome for everyone.
 
-| Name | Role <!-- TODO: "Featured Artist outreach" isn't a role but I'm not sure how to write it as one... --> |
+| Name | Role <!-- TODO: "Featured Artist outreach"/"* support" isn't a role but I'm not sure how to write it as one... --> |
 | :-- | :-- |
 | ::{ flag=AU }:: **[peppy](https://osu.ppy.sh/users/2)** | **Project lead** |
 | ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Modding and mapping scene advisor |
-| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | World Cup organiser and administrator |
+| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | [World Cup](/wiki/Tournaments#official-world-cups) organiser and administrator |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | World Cup organiser and administrator |
-| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Project and community maintainer, Featured Artist outreach |
+| ::{ flag=US }:: [Death](https://osu.ppy.sh/users/3242450) | [Moderator](/wiki/People/Global_Moderation_Team), technical support |
+| ::{ flag=US }:: [Dntm8kmeeatu](https://osu.ppy.sh/users/5428812) | Moderator, technical support |
+| ::{ flag=SK }:: [Domco](https://osu.ppy.sh/users/3562660) | osu! developer |
+| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Project and community maintainer, [Featured Artist](/wiki/People/Featured_Artists) outreach |
 | ::{ flag=JP }:: [flyte](https://osu.ppy.sh/users/3103765) | Design lead, resident Creative Cloud watcher |
-| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Modding and mapping scene advisor |
+| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | osu! developer |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leader, [Beatmap Spotlights](/wiki/Beatmap_Spotlights) manager, radar's secretary |
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | World Cup organiser and administrator |
-| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Tournament support, Featured Artist outreach |
+| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Featured Artist outreach, World Cup secretary |
```

Comment:
I am not in favor of the "secretary" terms as it is ambiguous to what that even means. World Cup Organiser, while they may not be LeoFLT, Chillier, etc,  still means they help and organize them. I like the consistency of organizer for wiki purposes.

Reply:
mango suggested to describe it this way since he does mostly scheduling and keeping things moving on time, and organisation is not the main focus anyway

Reply:
ok sounds good